Script/AtomicEditor/editor/Editor.ts

```diff
@@ -34,12 +34,19 @@ class Editor extends Atomic.ScriptObject {
 
         Editor.instance = this;
 
+        Preferences.getInstance().read();
+
+        var editorWindow = Preferences.getInstance().editorWindow;
+
+        if (editorWindow) {
+            Atomic.graphics.setWindowSize(editorWindow.width, editorWindow.height);
+            Atomic.graphics.setWindowPosition(editorWindow.x, editorWindow.y);
+        }
```

Comment:
It would be good if the preferences for the window size and position were parsed at Editor start, and used during the creation of the editor window.  Otherwise, the editor window is created, then immediately resized and moved.